### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,99 @@
...
+#### Docker Hub
 
 ```bash
-DISABLE_E2E_TESTS=true make test
+docker pull inistcnrs/lodex
+docker run -it inistcnrs/lodex
 ```
 
-You can also [define this environment variable on Travis](https://docs.travis-ci.com/user/environment-variables/), for a specific build or the whole project.
+## Use full links
```
Comment (parmentf) [inline README.md]:
```suggestion
## Useful links
```

File: README.md [hunk 2]
```diff
@@ -1,92 +1,99 @@
-# [Lodex](http://lodex.inist.fr) &middot; [![Build Status](https://travis-ci.org/Inist-CNRS/lodex.svg?branch=master)](https://travis-ci.org/Inist-CNRS/lodex) [![bitHound Overall Score](https://cdn.rawgit.com/aleen42/badges/master/src/gitbook_1.svg)](https://lodex.gitbooks.io/lodex-user-documentation) [![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Lodex CI](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml/badge.svg)](https://github.com/Inist-CNRS/lodex/actions/workflows/lodex-ci.yml)
+[![Licence](https://img.shields.io/badge/licence-CeCILL%202.1-yellow.svg)](http://www.cecill.info)
+[![Documentation](https://img.shields.io/badge/Documentation-f48022)](https://lodex.inist.fr/docs/)
+![Docker Image Version (latest semver)](https://img.shields.io/docker/v/inistcnrs/lodex)
```
Comment (parmentf) [inline README.md]: Strange: this image has a link (to <https://camo.githubusercontent.com/9b2a32c713dbd8b402938134e8cc03295a39b23ddb8e64b1eb1ed419d6068061/68747470733a2f2f696d672e736869656c64732e696f2f646f636b65722f762f696e697374636e72732f6c6f646578>), but I don't understand how.

File: README.md [hunk 3]
```diff
@@ -1,92 +1,99 @@
...
+
+# [Lodex](http://lodex.inist.fr)
 
 <img src="https://user-images.githubusercontent.com/7420853/30152932-1794db3c-93b5-11e7-98ab-a7f28d0061cb.png" width=150 align=right>
```
Comment (parmentf) [inline README.md]: FYI: Maybe this logo will change soon.

File: README.md [hunk 4]
```diff
@@ -1,92 +1,99 @@
...
 
-Lodex is a tool to enable publishing a set of data `csv`, `tsv`, `xml`, `json`, ... in semantic web formats `JSON-LD`, `N-Quads`, ... and propose to manipulate them in a backoffice.
+Lodex is a tool facilitating the publication of a dataset in various formats, including 'csv', 'tsv', 'xml', 'json', among others.
```
Comment (parmentf) [inline README.md]: I don't like single quotes around words, but it's a personal taste.

File: README.md [hunk 5]
```diff
@@ -1,92 +1,99 @@
...
+The platform also offers features to manipulate the data in a back-office environment.
 
-To see what Lodex can do, check out <https://data.istex.fr/> or the user documentation at <https://lodex.gitbook.io/lodex-user-documentation/>
+To see what Lodex can do, please refer to the website at https://data.istex.fr/ or
```
Comment (parmentf) [inline README.md]:
```suggestion
To see what Lodex can do, please refer to the website at <https://data.istex.fr/> or 
```

Please use [markdownlint](https://github.com/DavidAnson/markdownlint#overview), to ensure Markdown will work on most Markdown engine (I know it works as is in GitHub, but you never know...).

File: README.md [hunk 6]
```diff
@@ -1,92 +1,99 @@
...
 
 <img src="https://user-images.githubusercontent.com/7420853/30152932-1794db3c-93b5-11e7-98ab-a7f28d0061cb.png" width=150 align=right>
 
-Lodex is a tool to enable publishing a set of data `csv`, `tsv`, `xml`, `json`, ... in semantic web formats `JSON-LD`, `N-Quads`, ... and propose to manipulate them in a backoffice.
+Lodex is a tool facilitating the publication of a dataset in various formats, including 'csv', 'tsv', 'xml', 'json', among others.
+The platform also offers features to manipulate the data in a back-office environment.
 
-To see what Lodex can do, check out <https://data.istex.fr/> or the user documentation at <https://lodex.gitbook.io/lodex-user-documentation/>
+To see what Lodex can do, please refer to the website at https://data.istex.fr/ or 
+consult the user documentation available at https://lodex.inist.fr/docs/.
```
Comment (parmentf) [inline README.md]:
```suggestion
consult the user documentation available at <https://lodex.inist.fr/docs/>.
```

File: README.md [hunk 7]
```diff
@@ -1,92 +1,99 @@
...
 ```bash
-npm install
-npm run build
-EZMASTER_MONGODB_HOST_PORT=localhost:27017 NODE_ENV=production npm start
+make run-dev
 ```
 
-NB : MongoDB should be started and listening on port 27017 (use e.g. `make
-mongo` to launch mongo).
+Go to http://localhost:3000/instances and log in using the credentials `root`/`secret`.
```
Comment (parmentf) [inline README.md]:
```suggestion
Go to <http://localhost:3000/instances> and log in using the credentials `root`/`secret`.
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AlasDiablo) [reply README.md]: I forgot to add the link to docker hub...

**Review by parmentf — APPROVED** [review]

**Pull request merged**